src/odemis/driver/pigcs.py

```diff
@@ -1971,6 +1967,25 @@ def __init__(self, busacc, address=None, axes=None, auto_suspend=10, suspend_mod
             self._encoder_mng[a] = t
             t.start()
 
+            # TODO: moved ref at init to Bus class for now as a hack to work around the below mentioned problem
+            # TODO: Check if there was any need to send the RON command via SetReferenceMode in the startServo method.
+            #  @Eric was there any reason for this? It seems to work fine in the tests without.
```

Comment:
IIRC, turning off the servo automatically sets the referenced state in the controller to unreferenced (which is fair, as from there on, the hardware might physically move without the controller noticing, which would cause a shift in the referencing position). So that's why every time we restart the servo we have to convince the hardware that it's referenced (in relative moves). That said, now, we also support suspending with just setting the PID to 0, and leaving the servo on. We could do this unconditionally if the hardware has been *really* referenced.